lib/features/debug/presentation/debug/debug_model.dart

```diff
@@ -6,23 +6,23 @@ import 'package:flutter_template/features/debug/presentation/debug/debug_screen.
 /// {@template debug_model.class}
 /// Model for [DebugScreen].
 /// {@endtemplate}
-final class DebugScreenModel extends BaseModel {
+final class DebugModel extends BaseModel {
   final IDebugRepository _debugRepository;
 
   /// {@macro debug_model.class}
-  DebugScreenModel({
+  DebugModel({
     required IDebugRepository debugRepository,
     required super.logWriter,
   }) : _debugRepository = debugRepository;
 
   /// Save server Url to local storage.
   Future<void> saveServerUrl(Url url) async {
-    await makeCall(() => _debugRepository.saveServerUrl(url));
+    final _ = await makeCall(() => _debugRepository.saveServerUrl(url));
```

Comment:
why do we need it? Because of unused return value warning?

Reply:
yes